### PR TITLE
better form method handling

### PR DESCRIPTION
**Pull request by mcarey1590**

### CODE REVIEW

File: iron-form.html [hunk 1]
```diff
@@ -48,7 +48,13 @@
     is: 'iron-form',
 
     extends: 'form',
-
+    
+    parameters: {
```
Comment (notwaldorf) [inline iron-form.html]: I think you meant `properties` :)

File: iron-form.html [hunk 2]
```diff
@@ -48,7 +48,13 @@
...
+      contentType: {
```
Comment (notwaldorf) [inline iron-form.html]: Could you also doc this property, please? (see iron-ajax, for example)

File: iron-form.html (hunk 1 above)
Comment (mcarey1590) [inline iron-form.html]: Hah, Indeed. I'll fix that and add documentation.

**Comment by mcarey1590** [comment]
fixed typo and added documentation

**Comment by notwaldorf** [comment]
Gah, one more thing, sorry. Would you mind squashing your commits before we merge? Thanks!

**Comment by mcarey1590** [comment]
done

**Comment by notwaldorf** [comment]
Thanks! :+1:

**Pull request merged**